int8 servo_id
bool enable